packages/react/src/components/Chip/Chip.mdx

```diff
@@ -20,9 +20,9 @@ import { Chip } from './index';
 
 Chip eksisterer i ulike varianter:
 
-- `Chip` er et alternativ til `button`. Den brukes når handlingen(e) er direkte relatert til hovedinnholdet og når handlingen ikke krever like mye oppmerksomhet som vanlige knapper. Det kan for eksempel være å klikke på en Chip for så å bli sendt til en søkeresultatside med filtrering på denne kategorien.
-- `Chip` med `checkmark=true` er alternativer til vanlige `radiobuttons` og `checkboxer`. Brukes til filtrering av innhold og data, når det skal tilbys et valg mellom flere ulike opsjoner. Innholdet endres til å vise kun det som er valgt.
-- `Removable Chip` er valgte verdier som brukeren kan fjerne. For eksempel filter som har blitt aktivert i et søk som kan fjernes igjen, eller som har blitt valgt i en `multiselect`.
+- `Chip` er et alternativ til `Button`. Den brukes når handlingen(e) er direkte relatert til hovedinnholdet og når handlingen ikke krever like mye oppmerksomhet som vanlige knapper. Det kan for eksempel være å klikke på en Chip for så å bli sendt til en søkeresultatside med filtrering på denne kategorien.
+- `Chip` med `checkmark=true` er alternativer til vanlige radiobuttons og checkboxer. Brukes til filtrering av innhold og data, når det skal tilbys et valg mellom flere ulike opsjoner. Innholdet endres til å vise kun det som er valgt.
```

Comment:
Is this correct description? `checkmark` is only to choose if checkmark is displayed or not. `selected` prop is the one used for toggling display of selected state.

Reply:
hehe - nice catch! :)